crates/solidity/testing/perf/projects.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
I ran a quick test, and turns out that the deepest tree we have under `sourcify` is in `mainnet` (997 levels deep):

- https://vscode.blockscan.com/1/0x56681458E00CafE1206313D2D033946f458FDEfD

I wonder if that use case is useful for benchmarks?

Reply:
We take twice as much as solc and antlr (in the respective measurements, that is, full run or parse-only):

```
	"slang_parse_all_files_duration_deep_nesting_cooldogs": {
		"Duration": {
			"value": 1735.72
		}
	},
	"slang_init_bindings_graph_duration_deep_nesting_cooldogs": {
		"Duration": {
			"value": 3467.37
		}
	},
	"slang_resolve_all_bindings_duration_deep_nesting_cooldogs": {
		"Duration": {
			"value": 6168.62
		}
	},
	"slang_total_deep_nesting_cooldogs": {
		"Duration": {
			"value": 11371.71
		}
	},
	"antlr_build_ast_duration_deep_nesting_cooldogs": {
		"Duration": {
			"value": 877.98
		}
	},
	"solc_build_ast_duration_deep_nesting_cooldogs": {
		"Duration": {
			"value": 6806.83
		}
	}
```